src/rpc/rawtransaction.cpp

```diff
@@ -352,14 +358,25 @@ UniValue createrawtransaction(const JSONRPCRequest& request)
             + HelpExampleCli("createrawtransaction", "\"[{\\\"txid\\\":\\\"myid\\\",\\\"vout\\\":0}]\" \"{\\\"data\\\":\\\"00010203\\\"}\"")
```

Comment:
Need to add an example for the sorted format? 

Right now it only shows examples with the 'backwards compatible" format